### PR TITLE
Scala 2.13 support

### PR DESCRIPTION
**Pull request by iRevive**
This PR adds the support of Scala 2.13.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,15 +15,16 @@ jobs:
   include:
     # stage="test" if no stage is specified
     - env: TEST="build"
-      script: sbt headerCheck compile test
+      script: sbt ";headerCheck; + compile; + test"
     # run ci-release only if previous stages passed
     - stage: release
-      script: sbt ci-release
+      script: sbt + ci-release
```
Comment (tpolecat) [inline .travis.yml]: `ci-release` doesn't need `+`... it's a command that calls `+publishSigned` and so on.

File: build.sbt [hunk 2]
```diff
@@ -1,3 +1,7 @@
+lazy val scala211Version = "2.11.12"
+lazy val scala212Version = "2.12.10"
+lazy val scala213Version = "2.13.0"
```
Comment (tpolecat) [inline build.sbt]: 2.13.1

File: .travis.yml (hunk 1 above)
Comment (iRevive) [reply .travis.yml]: Good point. Done.

File: build.sbt (hunk 2 above)
Comment (iRevive) [reply build.sbt]: Currently, kind-projector doesn't support 2.13.1. 
Issue: https://github.com/typelevel/kind-projector/issues/116.

Should we wait for their release first?

**Pull request merged**